releasewarrior/templates/fennec/release.json.tmpl

```diff
@@ -18,8 +18,8 @@
             },
             {
                 "alias": "pushapk",
-                "description": "run pushapk (out of tree relpro)",
-                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/old-how-tos/relpro.md#3-push-to-releases-dir-mirrors",
+                "description": "run pushapk",
+                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/wiki/Push-to-Google-Play#what-to-do",
```

Comment:
@mozbhearsum actually, didn't you hit issues with this? I thought for <=58.x fennec, we still need to use "publish_fennec" flavour, not the new beta "ship_fennec" flavour.

iow - we do this for current fennec dot releases: https://github.com/mozilla-releng/releasewarrior-2.0/blob/10d9b968cfb1fd812dd3425c77e2638ba29ead4c/old-how-tos/fennec-temp-relpro.md#publish-release

I could be wrong

Reply:
Yeah, I think you're right. My changes to all of these templates assume we're using in-tree releases everywhere except ESR - so the release/RC changes are equally invalid. Perhaps we should wait until merge day to merge this patch?